src/Components/Product/ProductList/ProductList.js

```diff
@@ -1,19 +1,52 @@
-import axios from "axios";
-//import { useState, useEffect } from "react";
+import { Row, Col, Spinner } from "reactstrap";
+import { useState, useEffect } from "react";
+import "./ProductList.css";
 
-axios.get('http://localhost:3002/users')
-        .then(response => {
-            console.log(response);
-        });
-const ProductList = () => {
-    
-    return (
-        <div>
+function ProductList() {
+    const [error, setError] = useState(null);
+    const [isLoaded, setIsLoaded] = useState(false);
```

Comment:
The initial state could be `true`. So by default component will start with spinner until the API call gets resolved.